app/controllers/feeds_controller.rb

```diff
@@ -157,4 +158,8 @@ def nilified_feed_params
       apple_config_attributes: [:id, :publish_enabled, :sync_blocks_rss, {key_attributes: %i[id provider_id key_id key_pem_b64]}]
     )
   end
+
+  def exclude_default_episodes?
+    params[:feed][:exclude_default_episodes] == "1"
```

Comment:
[ActiveModel::Type::Boolean.new.cast](https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html) is also handy for this.